src/components/VoteButton.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react"
+import { patchUpVote } from "../utils/api"
+import '../.css/VoteButton.css'
+
+const VoteButton = ({article, setArticle}) => {
+    const [voteClicked, setVoteClicked] = useState(true)
+
+    const handleUpVotes = async (event) => {
+        event.preventDefault()
+        setVoteClicked(!voteClicked)
+        if (voteClicked) {
+            const updatedArticle = {...article, votes: article.votes +1}
+            setArticle(updatedArticle)
```

Comment:
This is not how we should be updating the state. Take a look at this section of the notes on how to update state using the previous value of said state:
https://notes.northcoders.com/courses/js-front-end/react-state#setstate-updater-functions